packages/launchpad/src/composables/index.ts

```diff
@@ -0,0 +1,7 @@
+import { computed } from 'vue'
```

Comment:
This is pretty neat but I wonder if it'll make it difficult to onboard people who are not familiar with Vue 🤔

Reply:
I think it makes it easier. The amount of boilerplate necessary for the new update:modelValue syntax is really irritating and I mess it up every time.

Reply:
I added some comments and a link to the article